packages/app/src/providers/hsds/hsds-api.ts

```diff
@@ -152,7 +151,7 @@ export class HsdsApi extends DataProviderApi {
     );
   }
 
-  public getExportURL<D extends Dataset<ArrayShape, DType>>(
+  public getExportURL<D extends Dataset<ArrayShape>>(
```

Comment:
VS Code added `DType` automatically when I implemented the method, but it's the default anyway.